hs-bindgen/app/hs-bindgen-cli.hs

```diff
@@ -53,7 +53,8 @@ execMode Cli{cliGlobalOpts=GlobalOpts{..}, ..} = case cliMode of
               , optsTracer         = tracer
               }
         translateCHeaders mu opts preprocessInputs
-      withTracer $ \tracer -> do
+      hsDecls <- maybe fatalError pure maybeHsDecls
+      maybeSuccess <- withTracer $ \tracer -> do
```

Comment:
This is suboptimal. Here, we create output, but creation of the prescriptive binding specifications may still fail.

Reply:
We discuss this in a separate conversation below.